rfd/xxxx-automatic-azure-server-discovery.md

```diff
@@ -0,0 +1,105 @@
+---
+authors: Andrew Burke (andrew.burke@goteleport.com)
+state: draft
+---
+
+# RFD X - Automatic discovery of Azure servers
+
+## What
+
+Teleport SSH services will be able to automatically discover Azure virtual machine
```

Comment:
From what I understood what will discover is the Discovery service and not the SSH.